.github/workflows/ios-test.yml

```diff
@@ -1,21 +1,23 @@
 name: Testing polyPod in iOS
 on: [push, pull_request]
 jobs:
-  test:
-    name: Testing polyPod iOS app
-    runs-on: macOS-10.15
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 16
-      - name: Lock xcode version
-        run: sudo xcode-select -switch /Applications/Xcode_12.4.app
-      - name: Build core and features
-        run: ./build.js
-      - name: Build iOS
-        run: |
-          cd ios
-          set -o pipefail
-          make test | xcpretty
+    test:
```

Comment:
Seems to just changing indentation if I get it right - is that actually related to this work? Doesn't seem so.

Reply:
It's adding another step, actually. The change of indentation if because I generally use emacs for this, although in this case I edited it using Code, which didn't like the previous indentation. Can bring it back to original, anyway.

Reply:
Would make it easier to understand what the change was! :D